### DISCUSSION TITLE
[Bug]: Can't read YAML values of objects (which dataview supports)

### DISCUSSION THREAD

**Discussion by SPiCaRiA** [RafaelGB/obsidian-db-folder#886] [Q&A] [answered]
### What happened?

Hi, thank you for making this great plug-in! However, I encountered some issues when I tried to add a column that is a member of an object if it is indented.

For example, I have such frontmatter setting in a note:

```yaml
problem:
  difficulty: "Easy"
  topic: "linear algebra"
```

DB folder could only detect the `problem` field as a whole but not the `difficulty` and `topic` fields. While in a Dataview query, we can use syntax like

```
TABLE
  problem.difficulty AS "Difficulty",
  problem.topic AS "Topic"
...
```

to access the members of an object.

### What platform were you using?

Desktop

### Version of the plugin

3.4.0

### Are you using the latest version of the plugin in your Obsidian vault?

- [X] I have verified that I am on the latest version

### Are you check if there is a similar issue?

- [X] I have verified that there are not similar issues

**Comment by RafaelGB** [comment, marked as answer]
hello @SPiCaRiA ,
please, check our documentation about nested properties. We support with some steps. https://rafaelgb.github.io/obsidian-db-folder/features/Properties/#nested-metadata
